src/baldr/connectivity_map.cc

```diff
@@ -70,16 +70,23 @@ json::MapPtr to_geometry(const polygon_t& polygon) {
   for (const auto& ring : polygon) {
     auto ring_coords = json::array({});
     for (const auto& coord : ring) {
-      // if (outer) {
+#if 1
```

Comment:
you left the declaration of `outer` commented out above, that said, im cool if we just delete this unused code. im sure i just had it in there for testing something and im not sure why i didnt remove it when i was done.. im sorry you're finding all the cobwebs :smile: